### PR TITLE
Fixed typos in overlays.md

### LINKED ISSUE #350: Typos in overlays.md
## Present condition
`capabilties`

## Proposed solution

`capabilities`

## Screenshot

Screenshot (263)

### PR DESCRIPTION
**Pull request by bishal7679**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

typos `capabilties` fixed now by this PR 

**Which issue(s) this PR fixes**:

## Fixes #350 

```release-note
  typos fixed by this PR in overlays.md
```

### CODE REVIEW

File: docs/release/overlays.md [hunk 1]
```diff
@@ -189,4 +189,4 @@ When it comes to creation of overlays there are no strict rules on how overlays
 
     2. If a critical bug fix involves multiple components (driver and sidecar), cherry-pick all the changes to the older release branch.
 
-   After a change has been cherry-picked to a release branch, a new patch release will have to be cut on that branch, and the overlays would be updated to reflect the new driver image version.
\ No newline at end of file
+   After a change has been cherry-picked to a release branch, a new patch release will have to be cut on that branch, and the overlays would be updated to reflect the new driver image version.
```
Comment (leiyiz) [inline docs/release/overlays.md:192]: I don't see the change here

Comment (bishal7679) [reply docs/release/overlays.md:192]: Actually It is taking automatically
I was trying to make a new PR but the same thing is happening
I dont know what is the reason here

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve

**Comment by bishal7679** [comment]
/ok-to-test

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by bishal7679** [comment]
@davidz627 @jingxu97 @leiyiz 
Could you proceed for merging this PR?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by bishal7679** [comment]
/retest-required

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request merged**